.github/workflows/tests-regression.yaml

```diff
@@ -0,0 +1,74 @@
+name: ETL Pipeline Tests (Regression)
+
+on: 
+  push:
+    branches: [ main ]
+  schedule:
+    - cron:  '0 0 * * 1'
+
+jobs:
+  changelog:
+    name: Updates Changelog
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: dangoslen/changelog-enforcer@v1.4.0
+        with:
+          changeLogPath: "CHANGELOG.md"
+          skipLabel: "no-changelog"
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+      - name: Cache dependencies
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('requirements.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+            ${{ runner.os }}-
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r dev-requirements.txt
+          pip install -r requirements.txt
+          pip install allure-pytest
```

Comment:
Since you've added the option to run this through the Makefile (which, thanks!) can you add this to `dev-requirements.txt`? Just so anyone cloning this in the future doesn't end up wondering why they can't get it to work locally

Reply:
I can do that. I also need to refactor my GitHub workflow to include the make command that runs allure-test. I'll do that in the next few commits.